lib/register.js

```diff
@@ -23,10 +34,11 @@ function handler({logger}) {
 async function register(req, res) {
   const registrar = req.srf.locals.registrar;
   const registration = req.registration;
+  const aor = registration.aor;
   let expires = registration.expires;
-  let contact = req.getParsedHeader('Contact')[0].uri;
+  let contact = registration.contact;
   let contactHdr = req.get('Contact');
-  const protocol = isWSS(req) ? 'wss' : req.protocol;
```

Comment:
We should probably change/fix this `isWSS` method, because it doesn't really handle the case where the protocol is 'ws', ie unsecured websockets.   The root problem is maybe that `req.protocol` isn't sophisticated enough, and is returning 'tcp' when the protocol is really 'wss'.  I should fix that in drachtio-srf, but maybe for now we should change this method to be `getSipProtocol(req)` or something, and returns one of 'udp','tcp','wss', or 'ws'